### PR TITLE
⭐️ Add Sophos Endpoint Defense to EDR Policy

### PR DESCRIPTION
**Pull request by tomtrix**
Dear Mondoo team,

hope you had a good start into this new week!

One of our valued customers requested support to detect, if Sophos Endpoint Defense is installed at their systems.

To achieve this, I've extended the relating ```core/mondoo-edr-policy.mql.yaml```.

Have a nice day and looking forward to your merge!

Kind regards
Tom

Tom Kretschmann
System Engineer DevOps
SVA System Vertrieb Alexander

### CODE REVIEW

**Comment by tomtrix** [comment]
I'm validating the package names with our customers at the moment.

*Please wait for merge until I've removed the ```WIP``` from the title. Thanks for understanding! <3*

**Comment by chris-rock** [comment]
@tomtrix Nice addition. Let us know when its ready and we are happy to add this to the policy.

File: core/mondoo-edr-policy.mql.yaml [hunk 1]
```diff
@@ -236,3 +238,13 @@ queries:
     mql: |
       service('WazuhSvc').running
       service('WazuhSvc').enabled
+  - uid: mondoo-edr-policy-ensure-sophos-endpoint-defense-is-running-windows
+    title: Ensure Sophos Endpoint Defense Agent is running
+    filters: |
+      asset.family.contains('windows')
```
Comment (chris-rock) [inline core/mondoo-edr-policy.mql.yaml:244]:
```suggestion
      asset.family.contains('windows') &&
```

**Review by atomic111 — APPROVED** [review]
@tomtrix thank you for adding Sophos support

**Pull request merged**